### PR TITLE
Add comment TableSchema::class.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Issue | https://github.com/yiisoft/db/issues/246

### CODE REVIEW

File: src/TableSchema.php [hunk 1]
```diff
@@ -13,8 +13,14 @@
  */
 final class TableSchema extends AbstractTableSchema
 {
+    private ?string $comment = null;
     private array $foreignKeys = [];
 
+    public function getComment(): ?string
```
Comment (vjik) [inline src/TableSchema.php:19]: Suggest add this method to table schema in `yiisoft/db` that return null by default.

Comment (terabytesoftw) [reply src/TableSchema.php:19]: I prefer to do it in every implementation.

Comment (vjik) [reply src/TableSchema.php:19]: But in application I'm work with `TableSchema` from `yiisoft/db` and IDE don't see this methods.

Comment (samdark) [reply src/TableSchema.php:19]: It should be part of the interface, not a specific method of implementations.

**Pull request closed without merging**